Standards/scs-xxxx-v1-provider-network-standard.md

```diff
@@ -0,0 +1,200 @@
+| Subnet | Subdivision of available IP address space using address prefixes. In OpenStack also an abstraction for controlling IP address allocation in a virtual network. |
+| DHCP | Dynamic Host Configuration Protocol: Used to automatically configure hosts in a network with IP addresses, default routes, and other information such as DNS servers. |
+| Prefix | IP address prefix of a given bit-length N, written _/N_. Divides addresses into a network and a host part, a shorter prefix allows more hosts but takes up more address space. |
+| NAT | Network Address Translation: mapping (usually public) IPv4 addresses onto a different (usually private) address space. May allows multiple hosts to share a public address by multiplexing TCP/UDP ports. |
+| RBAC | Role-based Access Control: A mechanism in the Network API to give projects limited access to resources owned by other projects. Typically used by CSPs to create provider networks. |
+| Shared Network | Virtual network that is shared between projects in a way that allows direct attachment of servers. |
+| External Network | Virtual network that is shared between projects in a way that only allows virtual routers to use it as external gateway. Typically used by CSPs to provide access to networks outside of the cloud environment. |
+| Provider Network | Any shared or external network that is managed by the CSP. |
+
+## Motivation
+
+The ability to interface virtualised resources with networks outside of the cloud environment, such as the internet, is an integral part of IaaS.
+Providing external access in an OpenStack cloud requires a number of configuration choices from the CSP, some of which have direct implications on how users interact with the cloud.
+This standard identifies some of these options and defines a baseline setup that provides flexibility and consistency to users but is also realistic to implement for CSPs.
+
+## Design Considerations
+
+This section will provide some general background on OpenStack provider networks before considering specific options to standardize. 
+
+### Provider Network Access Control
+
+In OpenStack, ownership of resources is generally tracked through projects, and, as per default policy, only members of a project have access to its resources
+This is also true for CSP-managed resources, such as provider networks, which have to be created in a CSP-internal project, and are initially only accessible in this project.
+
+The Network API's Role Based Access Control (RBAC) extension can then be used to share it with other projects.
+RBAC rules for networks support the two actions `access_as_external` and `access_as_shared`, and can be created automatically on `openstack network create` with the options `--external` and `--share`.
+* `access_as_external` allows networks to be used as external gateway for virtual routers in the target projects. Such networks are in the following referred to as _external networks_.
```

Comment:
Another notable thing `access_as_external` allows is creation of floating IPs

Reply:
I have added a mention of that.